care/facility/api/viewsets/facility_users.py

```diff
@@ -16,6 +16,7 @@ class UserFilter(filters.FilterSet):
         choices=[(key, key) for key in User.TYPE_VALUE_MAP],
         coerce=lambda role: User.TYPE_VALUE_MAP[role],
     )
+    username = filters.CharFilter(field_name="username", lookup_expr="icontains")
```

Comment:
Doesn't line 35 already allow this?

<img width="624" alt="image" src="https://github.com/user-attachments/assets/422b6b8e-ece8-4a85-a5a0-5b1df67362ab">

Reply:
@rithviknishad Nah, that didn't work for some reason 🤔

Reply:
search fields will look for a search key and return anything that contains the term in any of the fields, filter works on one field only.

Reply:
> search fields will look for a search key and return anything that contains the term in any of the fields, filter works on one field only.

Ah alright, search_fields were not being applied to the query set; I added that to get_queryset and removed the filter and that works (FE only passes in username anyways, so gets the same result).